src/test-utilities/react-testing.tsx

```diff
@@ -22,8 +22,8 @@ interface Providers {
   themeProvider: ThemeProviderContextType;
   frame: FrameContextType;
 }
-type Options = {};
-type Context = DeepPartial<Providers>;
+type Options = DeepPartial<Providers>;
+type Context = Providers;
```

Comment:
This is provided to the render method of `createMount`. Which will currently be the provider contests.